references/cli/adopt.go

```diff
@@ -188,6 +189,20 @@ func (opt *AdoptOptions) Complete(f velacmd.Factory, cmd *cobra.Command, args []
 	} else {
 		opt.AdoptTemplate = defaultAdoptTemplate
 	}
+	if opt.AppName != "" {
+		app, err := f.AppGetter().Get(opt.AppName, opt.AppNamespace)
```

Comment:
check the function `loadRemoteApplication`

Reply:
I am currently encountering some difficulties with the implementation of the loadRemoteApplication function here! Could you help me out here!